### ISSUE TITLE
[Issue]: Jellyfin.Networking.Manager.NetworkManager looses external interfaces(caused by process hang during playback)

### ISSUE THREAD

**Issue by DunkelArts** [jellyfin/jellyfin#11639]
### Please describe your bug

When Playing media jellyfin randomly looses sight of all network interfaces (happens every 10 - 20minutes)
This cause all sessions to die.
Connection comes back up after 2 -  3 minutes

Also this seems to delete alll my sessions, and forces me to log in again every time the server comes back up

Tested with
-  Chrome
- Android
- Hardware Acceleration on / off

### Reproduction Steps

1. Install new Server
2. playback something

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS:Proxmox
- Linux Kernel: 6.5.13-5-pve
- Virtualization: Docker
- Clients: Browser, Android
- Browser: tested in Brave / Firefox
- FFmpeg Version: 6.0.1-Jellyfin
- Playback Method: Direct Play
- Hardware Acceleration: NVEC
- GPU Model: GTX1050Ti
- Plugins: Playback Reporting, Fanart
- Reverse Proxy: Njc21/nginx-proxy-manager:2.11.2
- Base URL: none
- Networking: docker NAT
- Storage: local
```

### Jellyfin logs

```shell
[20:28:54] [WRN] [461] Jellyfin.Networking.Manager.NetworkManager: 172.70.247.53: External request received, no matching external bind address found, trying internal addresses.
[20:28:54] [WRN] [461] Jellyfin.Networking.Manager.NetworkManager: 172.70.247.53: External request received, but no external interface found. Need to route through internal network.
[20:28:54] [WRN] [472] Jellyfin.Networking.Manager.NetworkManager: 172.70.243.132: External request received, no matching external bind address found, trying internal addresses.
[20:28:54] [WRN] [472] Jellyfin.Networking.Manager.NetworkManager: 172.70.243.132: External request received, but no external interface found. Need to route through internal network.
[20:28:55] [INF] [461] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for dunkelarts has succeeded.
[20:28:55] [INF] [461] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user dunkelarts: 3/10
[20:28:55] [INF] [461] Emby.Server.Implementations.Session.SessionManager: Logging out access token 5d3a363ffcbf4ae1885c063c2c339443
[20:28:55] [INF] [461] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user e3a2e528-22d2-48b5-b312-9aeb662fce62
[20:28:55] [INF] [539] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for dunkelarts has succeeded.
[20:28:55] [INF] [539] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user dunkelarts: 3/10
[20:28:55] [INF] [539] Emby.Server.Implementations.Session.SessionManager: Logging out access token 15e1997b3df94f0e8cbc4d59b548731d
[20:28:55] [INF] [539] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user e3a2e528-22d2-48b5-b312-9aeb662fce62
[20:30:31] [INF] [512] MediaBrowser.MediaEncoding.Transcoding.TranscodeManager: FFmpeg exited with code 0
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vin86** [comment]
same errore here

**Comment by felix920506** [comment]
Fixed in #11670
